src/ChluIPFS.js

```diff
@@ -297,22 +329,27 @@ class ChluIPFS {
      */
     async importDID(did, publish, waitForReplication) {
         await this.waitUntilReady()
-        return await this.did.import(did, publish, waitForReplication)
+        return await this.didIpfsHelper.import(did, publish, waitForReplication)
+    }
+
+    async getReviewsWrittenByDID(didId, offset, limit) {
+        await this.waitUntilReady()
+        return await this.orbitDb.getReviewsWrittenByDID(didId, offset, limit)
```

Comment:
This is what will change in orbitdb-lite implementation right? As in, orbitdb-lite won't support this call. Basically, this call won't be ever made from the browser, so should we put a guard around it? Check env, if it is browser, then raise exception. What do you think?

Reply:
I think the proper solve here is that the implementation of getReviewsWrittenByDID in the Index will throw saying that this index does not implement it.

Then, from the wallet or whatever other frontend, we use the api client for this call.

Reply:
Yes agree. We want to leave that solve for a later PR?

Reply:
Yes, we will fix this when we implement the Chlu DB